pom.xml

```diff
@@ -56,7 +56,7 @@
         <snapshots.repository.url/>
         <reports.repository.url/>
         <!-- Must MANUALLY update this if the ddf/support project's version changes -->
-        <ddf.support.version>2.3.14</ddf.support.version>
+        <ddf.support.version>2.3.15-SNAPSHOT</ddf.support.version>
```

Comment:
❓Should we release `ddf-support` instead of having a dependency on a `SNAPSHOT` version?

Reply:
Yes. This won't release until then. It's just referencing the snapshot until the release of support is completed.